### PR TITLE
Configure openstack-ironic-python-agent CI jobs

### PR DESCRIPTION
**Pull request by elfosardo**
Also add the repo to OCP lifecycle and branch management

### CODE REVIEW

**Comment by elfosardo** [comment]
/pj-rehearse pull-ci-openshift-openstack-ironic-python-agent-release-4.11-unit

**Comment by elfosardo** [comment]
/pj-rehearse pull-ci-openshift-openstack-ironic-python-agent-release-4.11-unit

**Comment by elfosardo** [comment]
/pj-rehearse pull-ci-openshift-openstack-ironic-python-agent-release-4.11-unit

**Comment by elfosardo** [comment]
/pj-rehearse pull-ci-openshift-openstack-ironic-python-agent-release-4.11-unit

File: ci-operator/config/openshift/openstack-ironic-python-agent/OWNERS [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/openshift/openstack-sushy root OWNERS
```
Comment (dtantsur) [inline ci-operator/config/openshift/openstack-ironic-python-agent/OWNERS:2]: s/sushy/ironic-python-agent/

File: ci-operator/jobs/openshift/openstack-ironic-python-agent/OWNERS [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/openshift/openstack-sushy root OWNERS
```
Comment (dtantsur) [inline ci-operator/jobs/openshift/openstack-ironic-python-agent/OWNERS:2]: sushy again

**Pull request merged**